### PR TITLE
Bug 1314708: Clean up Tests

### PR DESCRIPTION
**Pull request by stephaniehobson**
- Check module imports
- Fix spacing
- Add destructive test definition
- Change test names
- Combine tests

### CODE REVIEW

**Comment by stephaniehobson** [comment]
@m8ttyB R?

File: tests/conftest.py [hunk 1]
```diff
@@ -11,10 +12,21 @@
...
+
+
+@pytest.fixture(scope='session')
+def sensitive_url(request):
+    # developer-local and localhost are not sensative
+    developer_local_match = str(request).startswith('https://developer-local')
+    localhost_match = str(request).startswith('localhost')
+    # treat it as sensitive if it's not identified as okay
+    #return not (developer_local_match or localhost_match)
+    return False
```
Comment (m8ttyB) [inline tests/conftest.py]: Is this a todo item? This will always return `False`.

File: tests/conftest.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import re
```
Comment (m8ttyB) [inline tests/conftest.py]: Is this import being used?

Comment (stephaniehobson) [inline tests/conftest.py]: It was being used when I was using a regex instead of string comparisons but it's not anymore.

**Comment by stephaniehobson** [comment]
Needs to be tested running on localhost. Going to install Docker and do that myself but if someone else spots this and is excited to try it let me know and I'll post steps.

**Pull request merged**